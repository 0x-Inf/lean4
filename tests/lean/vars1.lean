Import int.
Variables a b c : Int
Variables d b e : Int
Variables d e f : Int
Eval 1 + 1 + a + b + c + d + e + f