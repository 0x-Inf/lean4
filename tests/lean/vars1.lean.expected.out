  Set: pp::colors
  Set: pp::unicode
  Assumed: a
  Assumed: b
  Assumed: c
Error (line: 2, pos: 0) invalid object declaration, environment already has an object named 'b'
  Assumed: d
  Assumed: e
  Assumed: f
2 + a + b + c + d + e + f
